packages/http/httpcall_client.js

```diff
@@ -173,6 +173,19 @@ HTTP.call = function(method, url, options, callback) {
       }
     };
 
+    // Allow custom control over XHR and abort early.
+    if(options.beforeSend) {
```

Comment:
https://github.com/meteor/meteor/wiki/Meteor-Style-Guide: space after `if` (and a few lines down too)